hooks/mutate_pod.go

```diff
@@ -50,7 +51,7 @@ func (m *podMutator) Handle(ctx context.Context, req admission.Request) admissio
 	m.log.Info("mutating pod,", "name", namespacedName, "profile", m.profileName)
 
 	po := &corev1.Pod{}
-	err := m.decoder.Decode(req, po)
+	err := admission.NewDecoder(runtime.NewScheme()).Decode(req, po)
```

Comment:
Why do you create a new decoder?
Please use a `m.decoder`.

Reply:
@zoetrope 
Thank you for your kindness explanation that how do I change code at bozuman. I rewrite code which your appinted.